### PR TITLE
WIP my-joy-beta navigation and taxonomy

### PR DESCRIPTION
**Pull request by ramitos**

### CODE REVIEW

File: packages/joyent-boilerplate/.stylelintrc [hunk 1]
```diff
@@ -1,8 +1,3 @@
 {
-  "processors": ["stylelint-processor-styled-components"],
-  "extends": [
-    "stylelint-config-standard",
-    "stylelint-config-styled-components"
-  ],
-  "syntax": "scss"
+  "extends": ["stylelint-config-joyent-portal"]
```
Comment (SaraVieira) [inline packages/joyent-boilerplate/.stylelintrc]: Fancy 😱

File: packages/my-joy-beta/.eslintrc [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "joyent-portal",
+  "rules": {
+    "no-console": 0,
+    "new-cap": 0,
+    // temp
```
Comment (SaraVieira) [inline packages/my-joy-beta/.eslintrc]: Maybe remove this for now since we are starting from scratch

File: packages/my-joy-beta/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# joyent-boilerplate
```
Comment (SaraVieira) [inline packages/my-joy-beta/README.md]: Name is old

File: packages/ui-toolkit/package.json [hunk 4]
```diff
@@ -81,17 +77,15 @@
...
-      "styled-components/stylelint-processor-styled-components#2a33b5f",
+    "stylelint": "^8.1.1",
+    "stylelint-config-joyent-portal": "^1.0.1",
     "svgo": "^0.7.2",
     "tinycolor2": "^1.4.1",
     "title-case": "^2.1.1",
     "webpack": "^3.5.5"
   },
   "peerDependencies": {
+    "chart.js": "^2.7.0",
```
Comment (SaraVieira) [inline packages/ui-toolkit/package.json]: Why was this added ?

Comment (ramitos) [reply packages/ui-toolkit/package.json]: because it is needed by the metrics components

File: packages/my-joy-beta/README.md (hunk 3 above)
Comment (ramitos) [reply packages/my-joy-beta/README.md]: good catch, thanks 😉

File: packages/ui-toolkit/package.json (hunk 4 above)
Comment (SaraVieira) [reply packages/ui-toolkit/package.json]: Okay , carry on then :p

File: packages/joyent-boilerplate/.stylelintrc (hunk 1 above)
Comment (ramitos) [reply packages/joyent-boilerplate/.stylelintrc]: 😄
